sources/packages/backend/apps/api/src/route-controllers/student/student.institution.controller.ts

```diff
@@ -0,0 +1,58 @@
+import { Body, Controller, Injectable, Post } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { InstitutionService, StudentService } from "../../services";
+import { ClientTypeBaseRoute } from "../../types";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, UserToken } from "../../auth/decorators";
+import BaseController from "../BaseController";
+import {
+  StudentSearchAPIInDTO,
+  SearchStudentAPIOutDTO,
+} from "./models/student.dto";
+
+import { getISODateOnlyString } from "@sims/utilities";
+import { Student } from "@sims/sims-db";
+import { IInstitutionUserToken } from "../../auth";
+
+/**
+ * Student controller for institutions.
+ */
+@AllowAuthorizedParty(AuthorizedParties.institution)
+@Controller("student")
+@ApiTags(`${ClientTypeBaseRoute.Institution}-student`)
+@Injectable()
```

Comment:
Do we need a injectable here?

Reply:
When `@Controller` is used, `@Injectable` is not required.

Reply:
looks like its used in quite a few controllers, here are few 
![image](https://user-images.githubusercontent.com/77353155/230168471-a5e69c24-8834-41dc-bfe8-c0f8aaa27116.png)
![image](https://user-images.githubusercontent.com/77353155/230168553-b13a899a-4d2c-44d1-830a-ba072e1db5f2.png)
![image](https://user-images.githubusercontent.com/77353155/230168668-f59a87cb-874e-4331-806f-81ad16a06d99.png)

Reply:
I just moved this code, it already this way. Thanks!

Reply:
`@Controller` does not require the `@Injectable` because they are not injected anywhere. They were probably added by mistake someday and kept being added. Anyone that finds one can have them removed or create a PR to remove them all.

Reply:
> I just moved this code, it already this way. Thanks!

I did not get the part that states that this code was "moved". This is a brand-new file, right?
If the file is new and the code was copied, the statement "it already this way" does not seem accurate, IMO.

Reply:
It was already that way in the other file, I just moved the code from there to this one. I'm not saying that I should not have changed it and made it right. Sorry!